packages/orchestration/test/exos/cosmos-orchestration-account.test.ts

```diff
@@ -940,3 +941,54 @@ test('executeEncodedTx', async t => {
     'delegateMsgSuccess',
   );
 });
+
+test(`depositForBurn via Noble to Base`, async t => {
+  t.context.utils.populateChainHub();
+  const { chainHub } = t.context.facadeServices;
+  chainHub.registerChain('base', {
+    chainId: 'E8453',
+    cctpDestinationDomain: 0,
+  });
+  const makeTestCOAKit = prepareMakeTestCOAKit(t, t.context, { noble: true });
+  const nobleAccount = await makeTestCOAKit();
+  const amount = {
+    denom: 'uusdc',
+    value: 10n,
+  };
+
+  const actual = await E(nobleAccount).depositForBurn(
+    {
+      value: '0xe0d43135EBd2593907F8f56c25ADC1Bf94FCf993',
+      chainId: 'E8453',
+      // allegedName: 'base',
+      encoding: 'ethereum',
+    },
+    amount,
+  );
+
+  t.log('check the bridge');
+  t.deepEqual(actual, undefined);
+
+  const getAndDecodeLatestPacket = async () => {
+    await eventLoopIteration();
+    const { bridgeDowncalls } = await t.context.utils.inspectDibcBridge();
+    const latest = bridgeDowncalls[
+      bridgeDowncalls.length - 1
+    ] as IBCMethod<'sendPacket'>;
+    const { messages } = parseOutgoingTxPacket(latest.packet.data);
+    return MsgDepositForBurn.decode(messages[0].value);
+  };
+
+  const packet = await getAndDecodeLatestPacket();
+  t.log({ packet });
+  t.like(
+    packet,
+    {
+      amount: '10',
+      burnToken: 'uusdc',
+      destinationDomain: 0,
+      from: 'cosmos1test',
```

Comment:
Can we also show the `mintRecipient`? Might be helpful to confirm this is a bytes32 form of `0xe0d43135EBd2593907F8f56c25ADC1Bf94FCf993`

Reply:
Hmm. Do we think this is worthwhile?

```suggestion
      from: 'cosmos1test',
      mintRecipient: new Uint8Array([
        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
        0xe0, 0xd4, 0x31, 0x35, 0xeb, 0xd2, 0x59, 0x39, 0x07, 0xf8, 0xf5, 0x6c,
        0x25, 0xad, 0xc1, 0xbf, 0x94, 0xfc, 0xf9, 0x93,
      ]),
```

Reply:
@dckc looking for your input here - i know you're working on fixtures in tandem

Reply:
yes, it's worthwhile.

I can do it in a follow-on PR using the experimentally observed data.